crimethinc/website
# Replace TCE font icons with SVGs

# What are the relevant GitHub issues?

resolves [#1060](https://github.com/crimethinc/website/issues/1060)

# What does this pull request do?

The font icons in the top navigation of the TCE landing page have been replaced by svgs. They also have been resized to keep the alignment with their labels.

# How should this be manually tested?

Visit the TCE page, check that the icons in the top bar are properly labeled and coloured, and that alignment with the labels is respected on various screen sizes.

# Screenshots
## Mobile
![localhost_3000_tce](https://user-images.githubusercontent.com/76429737/105840428-4d9ff980-5fd3-11eb-868f-420cadff21ef.png)
## Tablet
![crimethinc com_tce (2)](https://user-images.githubusercontent.com/76429737/105840437-51cc1700-5fd3-11eb-989a-1d1a70470da7.png)
## Laptop
![crimethinc com_tce (1)](https://user-images.githubusercontent.com/76429737/105840452-55f83480-5fd3-11eb-99af-1623f92af060.png)
## Desktop
![crimethinc com_tce](https://user-images.githubusercontent.com/76429737/105840462-58f32500-5fd3-11eb-9029-540f28c6447f.png)
